auth_password_settings/__init__.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    Author: Dhaval Patel
+#    Copyright (C) 2011 - TODAY Denero Team. (<http://www.deneroteam.com>)
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as published
+#    by the Free Software Foundation, either version 3 of the License, or
+#    (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
```

Comment:
Header template has evolved to a shorter version
https://github.com/OCA/maintainer-tools/blob/master/template/module/exceptions.py#L1-L3

and the header in \_\_init\_\_.py is not mandatory because only import code and chars are only ascii